app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -169,27 +177,190 @@
 
     angular.extend(vm, {
       active: 'warlock',
+      activesets: '5/5/1',
+      progress: 0,
+      allSetTiers: [],
+      highestsets: {},
+      topsets: [],
+      lockeditems: { helmet: null, gauntlets: null, chest: null, leg: null, classItem: null, ghost: null, artifact: null },
       normalize: 335,
       doNormalize: false,
       type: 'Helmets',
       showBlues: false,
       showExotics: true,
       showYear1: false,
       combinations: null,
+      setOrder: '-str_val,-disc_val,-int_val',
+      setOrderValues: ['-str_val', '-disc_val', '-int_val'],
       statOrder: '-stats.STAT_INTELLECT.value',
       ranked: {},
       filter: {
         int: 3,
         dis: 2,
         str: 2
       },
+      lockedItemsValid: function(dropped_id, dropped_type) {
+          dropped_id = dropped_id.split('-')[1];
+          var item = _.findWhere(buckets[vm.active][dropped_type], {id: dropped_id});
+          var exoticCount = ((item.tier === 'Exotic')? 1 : 0);
+          for(var type in vm.lockeditems) {
+              var item = vm.lockeditems[type];
+              if(item === null || type === dropped_type) { continue; }
+              if(item.tier === 'Exotic' && item.type != 'classItem') {
```

Comment:
This should actually be `ClassItem` (capital C). wasn't able to equip my cape and another exotic otherwise.